values-changes.yaml

```diff
@@ -367,3 +367,6 @@ changes:
   - version: 41
     customFunctions:
       - installIstioHelmCharts
+  - version: 42
+    additions:
+      - otomi.useORCS: false
```

Comment:
I don't really see a good reason to use `additions` any longer. Adding this to defaults is enough in almost all use cases.